main.yml

```diff
@@ -3,15 +3,46 @@
   hosts: masters
   gather_facts: False
   vars:
-# Enable/Disable reconfigure for Kubevirt routes and oauth
+    # Enable/Disable reconfigure for Kubevirt routes and oauth
     kubevirt: "true"
+    hawkular: "true"
```

Comment:
@rut31337 can we use booleans vs strings.  Then a conditional is an easy `when: hawkular`

Reply:
that works for me